gravitee-apim-console-webui/src/management/api/analytics/pathMappings/api-path-mappings-add-dialog/api-path-mappings-add-dialog.component.html

```diff
@@ -0,0 +1,76 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<h2 mat-dialog-title class="title">Add Path Mapping</h2>
+
+<mat-dialog-content class="content">
+  <mat-tab-group dynamicHeight mat-stretch-tabs (selectedTabChange)="onSelectedTab($event)">
+    <!-- Add Path Mapping -->
+    <mat-tab [label]="tabLabels.AddPathMapping">
+      <div class="content__url-tab">
+        <mat-dialog-content class="content" [formGroup]="pathFormGroup">
+          <mat-form-field class="content__path__form-field" appearance="fill">
+            <mat-label>Path mapping</mat-label>
+            <input
+              type="text"
+              aria-label="Path mapping input"
+              placeholder="/products/:productId"
+              matInput
+              formControlName="path"
+              required
+            />
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('required')">Path mapping is required.</mat-error>
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('isUnique')">Path mapping already exists.</mat-error>
+            <mat-error *ngIf="pathFormGroup.controls.path.hasError('pattern')"
+              >Wrong path mapping pattern. Example: /products/:productId</mat-error
+            >
+          </mat-form-field>
+        </mat-dialog-content>
+      </div>
+    </mat-tab>
+
+    <!-- Import Swagger -->
+    <mat-tab [label]="tabLabels.SwaggerDocument">
+      <div class="content__url-tab">
+        <mat-dialog-content class="content" [formGroup]="pathFormGroup">
+          <mat-radio-group formControlName="path" class="radio-group" data-testid="designer-template-parameter-case">
```

Comment:
The first time I select a radio button I have the error telling  me that the path mapping doesn't respect the correct pattern is it the wanted behaviour ?

![Screenshot 2023-01-20 at 15 05 06](https://user-images.githubusercontent.com/25704259/213716149-cbeefffa-4a3e-4e0e-a7ad-66b3d5b67c2e.png)

Reply:
This is something that I wanted to talk about it. Do we want to have the same checks for both pages or not? The old page doesn't check